scripts/create_development_docker_compose.py

```diff
@@ -30,9 +30,18 @@ def process(self, data: Any) -> Any:
 
 
 class VolumeDevelopStrategy(YamlProcessor):
+    @staticmethod
+    def create_karton_service_list(data: Any) -> List[str]:
```

Comment:
this way it shouldn't be called create_karton_service_list if it returns web as well